exercises/2-attack-of-the-pipelines/README.md

```diff
@@ -567,7 +567,7 @@ Some of the key things to note:
         // Global Vars
         NAMESPACE_PREFIX="<YOUR_NAME>"
         GITLAB_DOMAIN = "<GITLAB_FQDN>"
-        GITLAB_PROJECT = "<GIT_USERNAME>"
+        GITLAB_PROJECT = "<GITLAB_PROJECT>"
```

Comment:
Actually, I think this is correct as-is .... although we could probably rename `GITLAB_PROJECT` to `GITLAB_USERNAME` ... i.e.: the environment variable `GITLAB_PROJECT` is used to compose the url for where the username is.

Reply:
yeah, true that - is should be GITLAB_USERNAME i think, i will update this